scripts/get_raw_sun_data.py

```diff
@@ -0,0 +1,81 @@
+############
+# Pull raw sun data using pvlib
+#
+# 2021-09-01
+# Peter Dudfield
+#
+# The data is about
+# - 1MB for a 2 of days, for ~2000 sites and takes about ~1 minutes
+# - 6MB for a 10 of days, for ~2000 sites and takes about ~1 minutes
+# - 252MB for a 365 of days, for ~2000 sites and takes about ~11 minutes (on a macbook pro)
+
+# Decide to just go for one year of data
+# on 1st Jan 2019 and 2020, the biggest differences was in elevation was 1 degree,
+# More investigation has been done (link), and happy difference is less than 1 degree,
+# Therefore, its ok good to use 1 year of data, for all the years
+############
+
+import logging
+from datetime import datetime
+from pathlib import Path
+import pandas as pd
+import os
+import nowcasting_dataset
+from nowcasting_dataset.data_sources.gsp.eso import get_gsp_metadata_from_eso
+from nowcasting_dataset.data_sources.sun.raw_data_load_save import (
+    save_to_zarr,
+    get_azimuth_and_elevation,
+)
+from nowcasting_dataset.geospatial import lat_lon_to_osgb
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.DEBUG)
+logging.getLogger("urllib3").setLevel(logging.WARNING)
+
+
+# set up
+BUCKET = Path("solar-pv-nowcasting-data")
```

Comment:
It would be good to generalise this so this script can work on GCP, AWS, or locally.  Maybe this script could read the relevant parts of a YAML config file?

This could wait until a later PR, if you'd prefer!

Reply:
ill add an issue

Reply:
https://github.com/openclimatefix/nowcasting_dataset/issues/186